addons/web/static/src/core/file_upload/file_upload_progress_bar.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="web.FileUploadProgressBar" owl="1">
+        <div class="o_file_upload_progress_bar">
```

Comment:
do you need all these custom classes? can't you do it with just plain bootstrap? also, if you do need to use these classes,  they should be defined in a css file next to this template then